T2Wil/REMS
# Redux config

### What does this PR do?
- This PR adds redux configuration to the project

### What are the completed tasks
- Add redux
- Add react-redux
- Add counter example

### How should this be manually tested?
- clone the repo `git clone https://github.com/T2Wil/REMS.git`
- enter the directory `cd REMS`
- enter the app `cd my-app`
- switch to this branch `git checkout ch-set-redux-173762559`
- run `npm run start`
- You should see button counter with a displaying redux state that changes as button click.

### Relevant stories
[Story link](https://www.pivotaltracker.com/story/show/173762559)
